src/test/java/com/crm/pages/ActivityStreamPage.java

```diff
@@ -0,0 +1,54 @@
+package com.crm.pages;
+
+import com.crm.utilities.Driver;
+import org.junit.Assert;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.time.Duration;
+
+public class ActivityStreamPage extends BasePage{
+
+    @FindBy(xpath = "//span[@class='feed-add-post-form-link-text']")
+    public WebElement moreButton;
+
+    @FindBy(xpath = "//span[@class='menu-popup-item-text'][text()='Appreciation']")
+    public WebElement appreciationButton;
+
+    @FindBy(xpath = "//span[@id='bx-b-uploadfile-blogPostForm']")
+    public WebElement uploadFilesButton;
+
+    @FindBy(xpath = "//input[@name='bxu_files[]']")
+    public WebElement uploadElement;
+
+    @FindBy(xpath =
+            "//span[text()='Insert in text']"
+    ) // table where elements
+    public WebElement webTable_InsertTextButton_FirstElement;
+
+    @FindBy(xpath = "//tbody[@class='diskuf-placeholder-tbody']//tr[1]//td[5]") //
+    public WebElement webTable_DeleteButton_FirstElement;
+
```

Comment:
The method I've created below "clicksAndCheckIfNoDisplayed" to process my two tests, where I should be adding uploaded files into the text, and where I should be deleting all the elements. In such way I had to decrease the implicit wait duration for a while, in order to do not wait for the element that has been already deleted. And after I've set the implicit duration back. And I had to "catch" the NoSuchElementException as there're no such elements after they have been removed in the while loop.